data/remnant/remnant ships.txt

```diff
@@ -1140,6 +1140,59 @@ ship "Puffin"
 
 
 
+ship "Smew"
+	sprite "ship/smew"
+	thumbnail "thumbnail/smew"
+	attributes
+		category "Fighter"
+		licenses
+			Remnant
+		"cost" 971000
+		"shields" 1200
+		"hull" 2800
+		"required crew" 1
+		"bunks" 8
+		"mass" 50
+		"drag" 1.75
+		"heat dissipation" .65
+		"fuel capacity" 100
+		"cargo space" 70
+		"outfit space" 92
+		"weapon capacity" 10
+		"engine capacity" 46
+		"ramscoop" 0.8
+		"shield generation" 0.48
+		"shield energy" 0.32
+		"hull repair rate" 0.9
+		"hull energy" 0.58
```

Comment:
```suggestion
		"shield generation" 0.20
		"shield energy" 0.1
		"hull repair rate" 0.6
		"hull energy" 0.3
		"force protection" -0.25
		"hull protection" 0.5
		"shield protection" 0.25
```

Yes, that's a negative force protection. A design to catch damaging forces and channel them into more force to push the lifeboat away even more seems like a fun addition.

Reply:
I'm pretty sure that will just amplify incoming hit force (and maybe firing force), not convert incoming damage to incoming hit force.
I don't really think this should be done. It is effectively the same as boosting the shield and hull capacities and not reducing the regeneration so far, which I also don't think should be done.

Reply:
OK, that was an interesting suggestion, an idea to think about. (hence not a "changes requested" thing).

No, it wouldn't convert damage, that's just lore for it. But most things with hit force also have damage, so it'd generally fit. And yeah, would make it not a great weapon platform for anything with recoil.

Reply:
Anyway, I'm fine leaving it as is.